movie.sh

```diff
@@ -1,55 +1,58 @@
 #!bin/bash
 
-function getrating()
+function add_style()
 {
-#Fetch tuple from the Omdb API
-curl -s  "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
-#Getting the Line with the information
-grep -o imdbRating=".*" mov.txt > submov.txt
-input="submov.txt"
-echo ""
-ctr=0
-rating=""
-#Extracting the rating. Since the format is XYZ= "x"
-#Track the opening and closing quotes to get x which is rating
-#ctr tracks number of " encountered
-while IFS= read -r -n1 var
-do
-  if [ $ctr -eq 2 ]
-  then
-        break
-  elif [ $var == '"' ]
-  then
-        ctr=$(( $ctr + 1 ))
-  elif [ $ctr -eq 1 ]
-  then
-        rating="$rating$var"
-  else
-        continue
-  fi
-done < "$input"
-rm mov.txt
-rm submov.txt
+        tput bold
+        tput setaf $2
+        echo $1
+        tput sgr 0
+        return 0
+}
+
+mkdir tmp
```

Comment:
We didn't mean creating a tmp directory. What we meant is that there is already a directory at `/tmp` for this purpose. You can use that.